README.md

```
@@ -1,40 +1,256 @@
+**발표 자료:** [이천조(A팀) 프로젝트 발표 자료](https://www.canva.com/design/DAGU9REKjyE/EzsBM3kU3aBkLQCEiB8fZg/view?utm_content=DAGU9REKjyE&utm_campaign=designshare&utm_medium=link&utm_source=editor)
+
+<br/>
+
+## 👏 팀원
+
+<table align="center">
+    <tbody>
+        <tr>
+            <td>
+                <a href="https://github.com/oceanlee-seoul">
+                    <img src="https://avatars.githubusercontent.com/oceanlee-seoul" width="100" height="100"/>
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)